inst/pages/multiassay_ordination.qmd

```diff
@@ -37,18 +37,6 @@ data(HintikkaXOData, package = "mia")
 mae <- HintikkaXOData
 ```
 
-```{r MOFA2, message=FALSE, warning=FALSE}
-library(MOFA2)
-
-# For inter-operability between Python and R, and setting Python dependencies,
-# reticulate package is needed
-library(reticulate)
-# Let us assume that these have been installed already.
-reticulate::install_miniconda(force = TRUE)
-reticulate::use_miniconda(condaenv = "env1", required = FALSE)
-reticulate::py_install(packages = c("mofapy2"), pip = TRUE, python_version=3.6)
-```
-
```

Comment:
Does the workflow run if you remove these? At least MOFA2 should be loaded (others might be ok to remove)

Reply:
mofa2 is loaded later where it is used.